src/web_app/js/call_test.js

```diff
@@ -0,0 +1,58 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
+ *
+ *  Use of this source code is governed by a BSD-style license
+ *  that can be found in the LICENSE file in the root of the source
+ *  tree.
+ */
+
+/* More information about these options at jshint.com/docs/options */
+
+/* globals TestCase, SignalingChannel:true, requestUserMedia:true,
+   assertEquals, assertTrue */
+
+'use strict';
+
+var MockSignalingChannel = function() {
+};
+
+MockSignalingChannel.prototype.open = function() {
+  return new Promise(function(resolve) { resolve(); });
```

Comment:
Sure, I had that originally, I think I changed it while trying to debug the Promise.all issue.